ui/webui/src/users/UserAccount.jsx

```diff
@@ -35,7 +84,14 @@ export const UserAccount = () => {
               title='Create user'
             />
             <PageSection>
-                Not implemented
+                <Form isHorizontal>
+                    <FormGroup label='Real name'>
+                        <TextInput id='gecos' onChange={onGecosChange} placeholder='John Doe' />
+                    </FormGroup>
+                    <FormGroup label='User name'>
```

Comment:
`FormGroup` component has a required property called 'fieldId'. This should be the `id` of the field that is nested in it, in this case 'name'. The reason for this is accessibility of the html code, https://developer.mozilla.org/en-US/docs/Learn/Accessibility/What_is_accessibility

How this increases accessibility?
It adds an `for` attribute to the DOM label element, making the `label` point to the `input`. This has also another effect, that when the user will then click on the label, the input box will get focused, which is the desired behavior.

If you want to know more let me know and I can direct you to the correct documentation.